pkg/cmd/update/password.go

```diff
@@ -0,0 +1,91 @@
+package update
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+
+	"github.com/litmuschaos/litmusctl/pkg/apis"
+	"github.com/litmuschaos/litmusctl/pkg/types"
+	"github.com/litmuschaos/litmusctl/pkg/utils"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var PasswordCmd = &cobra.Command{
+	Use: "password",
+	Short: `Updates an account's password.
+		Examples(s)
+		#update a user's password
+		litmusctl update password
+		`,
+	Run: func(cmd *cobra.Command, args []string) {
+		var (
+			updatePasswordRequest types.UpdatePasswordInput
+		)
+
+		credentials, err := utils.GetCredentials(cmd)
+		utils.PrintError(err)
+
+		promptUsername := promptui.Prompt{
+			Label: "Username",
+		}
+		updatePasswordRequest.Username, err = promptUsername.Run()
+		utils.PrintError(err)
```

Comment:
Same as above, please check if err != nil condition wherever printing err. Please update in all other places